es/criteria/make-contributing-easy.md

```diff
@@ -22,18 +22,18 @@ order: 5
 
 * Garantizar que otros reutilicen la codebase.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"